_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,89 @@
+## Android setup
```

Comment:
Why do we need a separate copy of the whole android file for Linux? Can't we just include in the one extra KVM step?